dGame/dBehaviors/BasicAttackBehavior.cpp

```diff
@@ -81,8 +81,7 @@ void BasicAttackBehavior::DoHandleBehavior(BehaviorContext* context, RakNet::Bit
 	}
 
 	if (isImmune) {
-		LOG("Target targetEntity %llu is immune! (Is this method even used?)",branch.target); //Immune is succesfully proc'd
-		//Seriously, is this ever used? I never see it in the console.
+		LOG_DEBUG("Target targetEntity %llu is immune!", branch.target); //Immune is succesfully proc'd
```

Comment:
The log and comment are the same, so only one is necessary, I'd keep the log but ill leave that up to you to decide